docs/dictionary/function/libUrlMultipartFormAddPart.lcdoc

```diff
@@ -24,22 +24,22 @@ Example:
 put empty into tForm
 put "dave" into tName
 put "hello" into tMessage
-if libURLMultipartFormData (tForm, "name", tName, "message", tMessage) \
-     is not empty then
+put libURLMultipartFormData (tForm, "name", tName, "message", tMessage) into tError
+if tError is not empty then
    ##handle error and exit
 end if
 set the httpHeaders to line 1 of tForm
 delete line 1 of tForm
 put "&lt;file&gt;" & "C:/myfile.gif" into tFile
 put "image/gif" into tType
 put "binary" into tEnc
-if libURLMultipartFormAddPart(tForm,"file", tFile, tType, tEnc) \
-    is empty then
+put libURLMultipartFormAddPart(tForm,"file", tFile, tType, tEnc) into tError
+if tError is not empty then
    ##handle error and exit
```

Comment:
@livecodeali I've updated the example with a custom handler. I improved the handling of the httpheaders as well so that it is always the value it was prior to entering the command.